packages/adapters-library/src/adapters/aave-v3/products/a-token/aaveV3ATokenAdapter.ts

```diff
@@ -22,7 +23,6 @@ export class AaveV3ATokenPoolAdapter extends AaveBasePoolAdapter {
   adapterSettings = {
     enablePositionDetectionByProtocolTokenTransfer: true,
     includeInUnwrap: true,
-    version: 2,
```

Comment:
Dont need this differentiation as the declarators now add new value to the getProtocolTokens and buildMetadata methods